fail2ban/plan.sh

```diff
@@ -0,0 +1,47 @@
+pkg_name=fail2ban
+pkg_origin=core
+pkg_version="0.10.4"
+pkg_description="Daemon to ban hosts that cause multiple authentication errors."
+pkg_source="https://github.com/fail2ban/fail2ban/archive/${pkg_version}.tar.gz"
+pkg_shasum="d6ca1bbc7e7944f7acb2ba7c1065953cd9837680bc4d175f30ed155c6a372449"
+pkg_upstream_url="https://www.fail2ban.org"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_build_deps=(
+  core/python
+  core/coreutils
+)
+pkg_deps=(
+  core/iptables
+  core/sqlite
+)
+pkg_lib_dirs=(lib)
+pkg_bin_dirs=(bin)
+pkg_svc_user=root
```

Comment:
After I built the plan, i tried to execute `hab svc load <origin>/fail2ban`.  Habitat failed to load since the package is runnable.

```
[2][default:/src:0]# hab svc load bdangit/fail2ban
✗✗✗
✗✗✗ [Err: 0] hab-sup(UT)[components/sup/src/util/pkg.rs:103:17]: Package is not runnable: bdangit/fail2ban/0.10.4/20190208234426
✗✗✗
```

I believe you need to either call out `pkg_svc_run` or have a `hooks/run` fleshed out.